ds3-autogen-c/src/test/java/com/spectralogic/ds3autogen/c/CCodeGenerator_Test.java

```diff
@@ -51,22 +50,42 @@ public void testSingleDeleteRequestHandler() throws IOException, ParserException
 
         codeGenerator.generate(spec, fileUtils, Paths.get("/tmp"));
 
-        LOG.info("Generated code:\n" + new String(outputStream.toByteArray()));
+        final ByteArrayOutputStream bstream = (ByteArrayOutputStream) fileUtils.getOutputStream();
+        final String output = new String(bstream.toByteArray());
+
+        LOG.info("Generated code:\n" + output);
```

Comment:
We should start to add some unit tests to verify that the generated code is actually being generated correctly, instead of just being printed.